.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Task
+about: A specific piece of work
+title: '[Task] '
+type: 'Task'
+---
+
+# What is this task about?
+<!-- Briefly describe the task and what it aims to accomplish. -->
+
+
+# Why is this task needed?
+<!-- Explain the motivation, whether it's routine upkeep, community engagement, documentation, etc. -->
+
+
+# Relevant resources or references
+<!-- Add links, files, or names of people that may help with this task. -->
+
+
+# How can this issue be closed?
+<!-- State what needs to be done for this to be considered complete. Include any related tags or project boards. -->
+
```

Comment:
I know we can assign issues to folks, but do we want to have a section of the task template for related people?

Reply:
I'm not too sure about if folks outside the org can assign others to an issue. I know my habit is to tag involved folks in issues if it is relevant, like if we had an offline conversation about something. Is that a common tendency that would make a section on related people moot?

Reply:
I am also in this habit (or, I try to be), but we can't expect everyone to have that. Maybe we should take a poll in the slack to get more opinions?